frontend/src/lib/components/groups-page/NameTableError.svelte

```diff
@@ -21,6 +23,8 @@
 				return n === null || n === undefined || n.length === 0;
 			case GroupError.NameNonUnique:
 				return groups.some((g) => g === n);
+			case GroupError.NameInvalid:
+				return n.match(/^[\w -]+$/) === null;
```

Comment:
is it using unicode though? I think this one will only allow asci letters and not polish for example